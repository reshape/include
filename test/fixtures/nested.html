<div class="something">
  <include src="partials/nested.html"></include>
</div>
