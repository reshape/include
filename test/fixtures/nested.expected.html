<div class="something">
  <div class="container">
  <div class="button"><div class="button__text">Text</div></div>

</div>

</div>
